products/src/main/java/de/baleipzig/products/ProductController.java

```diff
@@ -0,0 +1,45 @@
+package de.baleipzig.products;
+
+import de.baleipzig.products.persistance.Product;
+import de.baleipzig.products.persistance.ProductDatabaseDummy;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+// Controller sitzt auf Port 80
+@RestController
+public class ProductController {
+
+    private final ProductDatabaseDummy databaseDummy;
```

Comment:
Warum hast du hier nicht das Interface benutzt? Der Controller sollte sich nicht interessieren wie genau die API erfüllt wird

Reply:
Der Name ist auch nicht so treffend

Reply:
Macht ein Interface hier nicht nur sinn wenn man meherere "Controller" in dem Service hat  ? 
ich hab das jz mal auf JPA umgemünzt, aber mir ergibt sich noch nicht warum ich ein Interface für meine REST API brauche :3